packages/scripts/utils/env.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * External dependencies
+ */
+const { isPlainObject } = require( 'lodash' );
+const request = require( 'request' );
+const DecompressZip = require( 'decompress-zip' );
+const chalk = require( 'chalk' );
+const { sprintf } = require( 'sprintf-js' );
+
+/**
+ * Node dependencies.
+ */
+const { execSync } = require( 'child_process' );
+const { env, exit, stdout } = require( 'process' );
+const { normalize } = require( 'path' );
+const { createWriteStream } = require( 'fs' );
+const { tmpdir } = require( 'os' );
+
+/**
+ * Merges two YAML configs together.
+ *
+ * All new data from newConfig will be added to originalConfig. When arrays in newConfig look like lists of volume
+ * mount instructions, it will attempt to replace items that mount from the same location. This allows the config
+ * to be safely updated, and it'll be reflected in the updated config.
+ *
+ * @param {Object} originalConfig The original config object that we're overwriting.
+ * @param {Object} newConfig      A new config object to merge into originalConfig.
+ * @param {string} baseDir        The base directory of the plugin, volumes in this directory belong to the plugin.
+ * @return {Object} The merged config object.
+ */
+function mergeYamlConfigs( originalConfig, newConfig, baseDir ) {
```

Comment:
YAML is an acronym so this should be `mergeYAMLConfigs`. (I know, I know.)

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#abbreviations-and-acronyms

Reply:
🙃 4d9bde963